spring-cloud-dataflow-shell/src/main/java/org/springframework/cloud/dataflow/shell/command/ModuleCommands.java

```diff
@@ -61,14 +62,18 @@
 
 	private static final String REGISTER_MODULE = "module register";
 
+	private static final String IMPORT_APPS = "apps import";
```

Comment:
When we rename `module` to `app`, we'll have commands that start with `app` and others that start with `apps`, which is not ideal for completion...